### ISSUE TITLE
Unable to use ownCloud after upgrading from v8.0.4 to v8.1 

### ISSUE THREAD

**Issue by j-ed** [owncloud/core#17457]
## Expected behavior
The upgrade from ownCloud v8.0.4 to v8.1 should be run without a problem.

## Actual behavior
After ugrading ownCloud from v8.0.4 to v8.1 and running `./occ upgrade` from the command line the following messages were shown on the console:
```
...
Disabled 3rd-party app: bookmarks
Disabled incompatible app: calendar
Disabled 3rd-party app: calendar
Disabled 3rd-party app: chat
Disabled incompatible app: contacts
Disabled 3rd-party app: contacts
Disabled 3rd-party app: files_mv
Disabled 3rd-party app: mozilla_sync
Disabled 3rd-party app: news
Disabled 3rd-party app: ownnote
Disabled 3rd-party app: tasks
Updating <updater> ...
Updated <updater> to 0.6
Updating <activity> ...
Updated <activity> to 2.0.1
Updating <files_sharing> ...
Updated <files_sharing> to 0.6.2
Updating <files_trashbin> ...
Updated <files_trashbin> to 0.6.  -> is 0.6.3
...
```
Unfortunately a login to ownCloud is not possible anymore, instead the following upgrade message is still shown:
![20150707_oc81_upgrade_message-1](https://cloud.githubusercontent.com/assets/2572764/8553197/0f6e29ca-24e2-11e5-8de3-67f579d80332.jpg)

After pressing the `Aktualisieren starten` button the following screen is instantly shown but nothing happens with the upgrade itself:
![20150707_oc81_upgrade_message-2](https://cloud.githubusercontent.com/assets/2572764/8553201/16fbed9e-24e2-11e5-9993-7ac4110844df.jpg)

Further analysis shows that something seems to fail with the update of the updater app. Although the following message has been shown during the update:
```
Updating <updater> ...
Updated <updater> to 0.6
```
The database still shows v0.4 installed on the machine:
![20150707_oc81_upgrade_db_versions](https://cloud.githubusercontent.com/assets/2572764/8553214/28007ae2-24e2-11e5-8efa-7f1e242b6678.jpg)

The assumption that the update app upgrade fails is confirmed by running the upgrade command again:
```
> ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
Turned on maintenance mode
Checked database schema update
Checked database schema update for apps
Updated database
Updating <updater> ...
Updated <updater> to 0.6
Update successful
Turned off maintenance mode
```

How could this problem be solved? BTW: the file `/updater/appinfo/info.xml` contains the version string `<version>0.5</version>` but the file `/updater/appinfo/version` contains `0.6`!

#### Server Configuration
OS: Linux 3.2.67
Web server: Apache2 2.4.12
Database: MySQL 5.1.73
PHP version: 5.4.42
ownCloud version: 8.1.0

#### Client Configuration
Browser: Firefox 39.0
Operating system: Windows 7

**Comment by RealRancor** [comment]
Mhhh, have seen a similar "update" loop reported in here with OC 8.0.x but can't find it at the moment. Will link it later if possible.

**Comment by j-ed** [comment]
I was able to get rid of the message by setting the version number in `/updater/appinfo/version` to `0.4`. Nevertheless that seems not to have solved all the problems. After upgrading the calendar and contacts app and enabling one app after the other via the command line interface my mobile clients are able to access ownCloud but I'm unable to access it via the WebGUI.

**Comment by oeves** [comment]
Same issue here. At the moment changing the version file solved the issue.

Here is the log from the occ updater:

Turned on maintenance mode
Checked database schema update
Checked database schema update for apps
Updated database
Disabled 3rd-party app: calendar
Disabled 3rd-party app: contacts
Updating <updater> ...
Updated <updater> to 0.6
Updating <activity> ...
Updated <activity> to 2.0.1
Updating <files_sharing> ...
Updated <files_sharing> to 0.6.2
Updating <files_trashbin> ...
Updated <files_trashbin> to 0.6.3
Updating <files_versions> ...
Updated <files_versions> to 1.0.6
Update 3rd-party app: calendar
Update 3rd-party app: contacts
Update successful
Turned off maintenance mode

**Comment by lachlan-00** [comment]
my 8.1 install was stuck in the same loop after upgrading from 8.0.4.

changing apps/updater/appinfo/version from 0.6 to 0.4 let me in right away

**Comment by zocker22** [comment]
That is really strange. I get the same error. In terminal I can repitedly enter "occ upgrade" and always get the same result:
```
php5.5-cli ./occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
Turned on maintenance mode
Checked database schema update
Checked database schema update for apps
Updated database
Updating <updater> ...
Updated <updater> to 0.6
Updating <user_webdavauth> ...
Updated <user_webdavauth> to 1.1.0.2
Updating <files_versions> ...
Updated <files_versions> to 1.0.6
Update successful
Turned off maintenance mode
```
``` 
php5.5-cli ./occ app:list
ownCloud or one of the apps require upgrade - only a limited number of commands are available
Enabled:
 - activity: 2.0.1
 - external: 1.2
 - files: 1.1.9
 - files_external: 0.2.3
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.2
 - files_texteditor: 0.4
 - files_trashbin: 0.6.3
 - files_videoviewer: 0.1.3
 - firstrunwizard: 1.1
Disabled:
 - encryption
 - files_locking
 - files_versions
 - gallery
 - provisioning_api
 - shorty
 - templateeditor
 - updater
 - user_external
 - user_ldap
 - user_webdavauth
```

I hope this helps somehow

**Comment by sorglos** [comment]
It's not a proper solution but I have changed the ```installed_version``` in the ```appconfig``` table to ```0.6``` to escape from the loop.

**Comment by harenber** [comment]
Suffering partly from the same. Upgrade stuck using the website but succeeded by CLI:

`root@cubie:/var/www/owncloud#  sudo -u www-data php occ upgrade
ownCloud is already latest version
root@cubie:/var/www/owncloud# `

clients (and WebDAV) work, but when trying to login through the WebGUI, it will silently re-open the login page no matter if the username/password combination is correct or now.

Chaning the `installed_version` of  `updater` to `0.6` did unfortunately not make any difference to me.

I even `apt-get purge`d the whole installation, removed the full `app` folder (only `data` and `config` left over), but that hasn't solved it.

The log files do not show any error, neither apache's nor owncloud's log.

Debian 8 running on ARM

**Comment by zocker22** [comment]
@harenber 
You need not only to change the version of the updater-app, but also from every app that has not properly been updated.
If you enter php5.5-cli ./occ upgrade and repeat the process once more. You will see all the apps that have not been upgraded properly. When you look into your database into the table *PRÄFIX*appconfig, you will see which version is currently installed. Either you change the value in the database or in the file /path/to/core/*APP-NAME*/appinfo/version. At the end both versions (DB an version-file) need to match. I recommend downgrading the version in the file. Then you don't have to change anything when upgrading the next time.

In my case I had to change the versions for
<updater> to 0.6
<user_webdavauth> to 1.1.0.2
<files_versions> to 1.0.6

I hope that helps

**Comment by Bugsbane** [comment]
I had this issue with the upgrader and the notes app. As with others here, downgrading the version numbers in the versino file fixed the issue for me.

**Comment by PVince81** [comment]
@j-ed said:

>How could this problem be solved? BTW: the file /updater/appinfo/info.xml contains the version string <version>0.5</version> but the file /updater/appinfo/version contains 0.6!

It is very likely to be what is causing the issue.
A quickfix would be to remove the "version" tag in info.xml and let the upgrade run again.

**Comment by PVince81** [comment]
Raised issue here: https://github.com/owncloud/core/issues/17598

**Comment by PVince81** [comment]
and https://github.com/owncloud/updater/pull/154 to fix the updater app

**Comment by PVince81** [comment]
@harenber try removing the version tag from "updater/appinfo/info.xml".

**Comment by harenber** [comment]
Am 13.07.15 um 10:38 schrieb Vincent Petry:
> @harenber <https://github.com/harenber> try removing the version tag
> from "updater/appinfo/info.xml".

Which info.xml is that?

It seems I don't have an app "updater" anymore after purging and
re-installing my OC:

root@cubie:/var/www/owncloud# find . -name info.xml
./apps/provisioning_api/appinfo/info.xml
./apps/user_external/appinfo/info.xml
./apps/files_external/appinfo/info.xml
./apps/user_webdavauth/appinfo/info.xml
./apps/files_pdfviewer/appinfo/info.xml
./apps/gallery/appinfo/info.xml
./apps/files_locking/appinfo/info.xml
./apps/files_videoviewer/appinfo/info.xml
./apps/files_versions/appinfo/info.xml
./apps/external/appinfo/info.xml
./apps/files_sharing/appinfo/info.xml
./apps/files_trashbin/appinfo/info.xml
./apps/encryption/appinfo/info.xml
./apps/files/appinfo/info.xml
./apps/templateeditor/appinfo/info.xml
./apps/files_texteditor/appinfo/info.xml
./apps/user_ldap/appinfo/info.xml
./apps/activity/appinfo/info.xml
./apps/firstrunwizard/appinfo/info.xml
root@cubie:/var/www/owncloud#

Thanks

  Torsten

**Comment by PVince81** [comment]
@harenber okay, so in your case the update loop is caused by something different. Can you compare the ownCloud version in "config.php" and the one in "version.php" ? They should match.

**Comment by harenber** [comment]
@PVince81 I think I really suffer from some other problem, as I don't have this update loop, I just cannot use the WebGUI (like @j-ed reported as well)

```bash
root@cubie:/var/www/owncloud#  sudo -u www-data php occ upgrade
ownCloud is already latest version
root@cubie:/var/www/owncloud#
```

here are the two version tags:

```bash
root@cubie:/var/www/owncloud# cat version.php
<?php
$OC_Version = array(8,1,0,8);
$OC_VersionString = '8.1.0';
$OC_Edition = '';
$OC_Channel = 'stable';
$OC_Build = '2015-07-02T22:33:50+00:00 06ec916b1128b9a1ec79751bdd78cc4a8c64ace8';
root@cubie:/var/www/owncloud# grep version config/config.php
  'version' => '8.1.0.8',
root@cubie:/var/www/owncloud#```

**Comment by PVince81** [comment]
@harenber With update loop I mean that the update page will reappear again.
If that's not the case for you, please raise your issues separately and use the issue template https://raw.githubusercontent.com/owncloud/core/master/issue_template.md so we can investigate what happened on your specific environment.

Thanks

**Comment by harenber** [comment]
@PVince81 Thanks !!! Did as advised.  #17600

**Comment by sorglos** [comment]
After the ```updater``` app (see above) I've had the same problem with ```calendar``` (0.7.1=>0.7.2). The only way to escape from the update loop  was deactivate and activate the app with the recommended intermediate step of refreshing the site before the activation (and set the allowed groups).

**Comment by dogsbody** [comment]
I have the same issue and now have an ownCloud I can't use :-/

Any help gratefully received.  Thank you.

```
# sudo -u stashadmin php occ upgrade
ownCloud or one of the apps require upgrade - only a limited number of commands are available
Turned on maintenance mode
Checked database schema update
Checked database schema update for apps
Updated database
Updating <updater> ...
Updated <updater> to 0.6
Updating <files_sharing> ...
Updated <files_sharing> to 0.6.2
Update successful
Turned off maintenance mode
```

**Comment by dogsbody** [comment]
Update, I have gotten things working again by running the following...
```
# sudo -u webuser php occ app:enable updater
# sudo -u webuser php occ app:enable files_sharing
# sudo -u webuser php occ app:list
# echo "0.4" > apps/updater/appinfo/version
# echo "0.6.1" > apps/files_sharing/appinfo/version
```
...are there any long term issues from this?

**Comment by harenber** [comment]
Thanks Dan for your information. Seems my installation suffers from a
different problem:

root@cubie:/var/www/owncloud# sudo -u www-data php occ app:enable updater
updater is already enabled
root@cubie:/var/www/owncloud# sudo -u www-data php occ app:enable
files_sharing
files_sharing is already enabled
root@cubie:/var/www/owncloud# sudo -u www-data php occ app:list
Enabled:
 - files: 1.1.9
 - files_external: 0.2.3
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.2
 - files_texteditor: 0.4
 - files_trashbin: 0.6.3
 - files_versions: 1.0.6
 - files_videoviewer: 0.1.3
 - firstrunwizard: 1.1
 - gallery: 0.6.0
Disabled:
 - activity
 - encryption
 - external
 - files_locking
 - provisioning_api
 - templateeditor
 - user_external
 - user_ldap
 - user_webdavauth

root@cubie:/var/www/owncloud# find . -name updater
root@cubie:/var/www/owncloud# echo "0.6.1" >
apps/files_sharing/appinfo/version
root@cubie:/var/www/owncloud#

But situation is still the same.

Meanwhile I think about installating a new instance of owncloud on a
fresh VM and copy all data manually through clients. A lot of work, but
I feel I cannot get the updated version to run again.

**Comment by PVince81** [comment]
Is anyone still stuck with a broken ownCloud 8.1 ?

8.1.3 should have the fixed updater app and hopefully upgrading should work now.

**Comment by j-ed** [comment]
@PVince81 The updater app problem doesn't appear on my server anymore after upgrading from v8.0.8 to v8.1.3, but I'm still not able to login.

**Comment by PVince81** [comment]
@j-ed anything new in the owncloud.log ?

**Comment by j-ed** [comment]
@PVince81 No, not really. Except the messages which are generated because the calendar and contacts app are not available, when mobile clients are trying to load it. I've now updated the bookmarks, calendar and contacts app to get rid of these messages.
These are the messages which are written to the log file from the moment I started the web browser, reloaded the login page and entered the login data:
```
### RELOAD LOGIN PAGE ###
{"reqId":"JR+bbld\/T6rANeKbl2eG","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `appid`, `configvalue` FROM `oc_appconfig` WHERE `configkey` = 'installed_version'","level":0,"time":"2015-09-22 13:59:21+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"JR+bbld\/T6rANeKbl2eG","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array ( )","level":0,"time":"2015-09-22 13:59:21+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"JR+bbld\/T6rANeKbl2eG","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:21+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"JR+bbld\/T6rANeKbl2eG","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:21+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"JR+bbld\/T6rANeKbl2eG","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:21+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `appid`, `configvalue` FROM `oc_appconfig` WHERE `configkey` = 'installed_version'","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array ( )","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `gid` FROM `oc_groups` WHERE `gid` = ?","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array (   0 => 'admin', )","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `uid`, `displayname` FROM `oc_users` WHERE LOWER(`uid`) = LOWER(?)","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"qKqFzN48REJiRRwwcnVJ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array (   0 => false, )","level":0,"time":"2015-09-22 13:59:22+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
### LOGIN ###
{"reqId":"A9AhdhjToWfBz0pIttQf","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `appid`, `configvalue` FROM `oc_appconfig` WHERE `configkey` = 'installed_version'","level":0,"time":"2015-09-22 14:07:13+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"A9AhdhjToWfBz0pIttQf","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array ( )","level":0,"time":"2015-09-22 14:07:13+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"A9AhdhjToWfBz0pIttQf","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:13+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"A9AhdhjToWfBz0pIttQf","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:13+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"A9AhdhjToWfBz0pIttQf","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:13+02:00","method":"POST","url":"\/owncloud\/"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `appid`, `configvalue` FROM `oc_appconfig` WHERE `configkey` = 'installed_version'","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array ( )","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `id` FROM `oc_jobs` WHERE `class` = ? AND `argument` = ?","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `gid` FROM `oc_groups` WHERE `gid` = ?","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array (   0 => 'admin', )","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB prepare : SELECT `uid`, `displayname` FROM `oc_users` WHERE LOWER(`uid`) = LOWER(?)","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
{"reqId":"UlI7bURge3YX9+9mkygQ","remoteAddr":"192.168.1.56","app":"core","message":"DB execute with arguments : array (   0 => false, )","level":0,"time":"2015-09-22 14:07:14+02:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=cc14b2d01bc618f211fc10716487bfba"}
### END LOGIN ###
```

**Comment by PVince81** [comment]
@j-ed I don't see anything wrong here.
What error message are you getting in the web UI when trying to login ? (screenshot?)
What error code are you getting when logging in through Webdav ? (cadaver or curl)

**Comment by j-ed** [comment]
@PVince81 No error messages were shown on the screen, just a reload of the login page took place in the web UI.
Nevertheless I think I've good news and found the root cause of the problem. I had the `log_query => true,` parameter in my configuration for debug purposes which definitely seems to have caused the problem.
After removing the parameter from the configuration file, a login was possible again without any problems.

After logging-in to ownCloud without that parameter set, I added the parameter and tried to log-off but the the following message was instantly shown:
```
{"data":{"message":"Token abgelaufen. Bitte lade die Seite neu.","error":"token_expired"},"status":"error"}
```
The login screen seems to suppress this message and didn't write any information to the log file although I assume that an expired token message was responsible for the problem.

**Comment by RealRancor** [comment]
@j-ed It might be useful that you stick on one issue instead of posting the infos in both. I think that makes it hard to follow if two issues are used in parallel.

**Comment by lspironello** [comment]
Hello.  I'm having a the same "loop" problem. Running Ubuntu 15.04.   Decided to use owncloud repo.  Upgrade from 8.0.3 -> 8.1.3.0 (owncloud repo version).   Attempted to login from a number of account with the respond of the login page being displayed again.  I've tried all of the methods listed above to resolve this issue without success.  I've disabled app, down to file (1.1.10), files_sharing(0.6.2) and updater (0.6).  OC is now effectively, unusable.  Ideas on other things to try?  Thank you.
